addons/crm_claim/crm_claim.py

```diff
@@ -38,13 +43,63 @@ class crm_claim(osv.osv):
     _name = "crm.claim"
     _description = "Claim"
     _order = "priority,date desc"
-    _inherit = ['mail.thread']
+    _inherit = ['mail.thread', 'ir.needaction_mixin']
 
     def _get_default_stage_id(self, cr, uid, context=None):
-        """ Gives default stage_id """
-        team_id = self.pool['crm.team']._get_default_team_id(cr, uid, context=context)
+        if context is None:
+            context = {}
+        team_id = context.get('default_team_id')
         return self.stage_find(cr, uid, [], team_id, [('sequence', '=', '1')], context=context)
 
+    def _read_group_stage_ids(self, cr, uid, ids, domain, read_group_order=None, access_rights_uid=None, context=None):
+        access_rights_uid = access_rights_uid or uid
+        Stage = self.pool['crm.claim.stage']
+        order = Stage._order
+        # lame hack to allow reverting search, should just work in the trivial case
+        if read_group_order == 'stage_id desc':
+            order = "%s desc" % order
+        # retrieve team_id from the context and write the domain
+        # - ('id', 'in', 'ids'): add columns that should be present
+        # - OR ('case_default', '=', True), ('fold', '=', False): add default columns that are not folded
+        # - OR ('team_ids', 'in', team_id), ('fold', '=', False) if team_id: add team columns that are not folded
+        search_domain = []
+        team_id = context.get('default_team_id')
+        if team_id:
+            search_domain += ['|', ('team_ids', '=', team_id), ('id', 'in', ids)]
+        # perform search
+        stage_ids = Stage._search(cr, uid, search_domain, order=order, access_rights_uid=access_rights_uid, context=context)
+        result = Stage.name_get(cr, access_rights_uid, stage_ids, context=context)
+        # restore order of the search
+        result.sort(lambda x,y: cmp(stage_ids.index(x[0]), stage_ids.index(y[0])))
+
+        fold = {}
+        for stage in Stage.browse(cr, access_rights_uid, stage_ids, context=context):
+            fold[stage.id] = stage.fold or False
+        return result, fold
+
+    def action_calendar_followup(self, cr, uid, ids, context=None):
+        action = self.pool.get('ir.actions.act_window').for_xml_id(cr, uid, 'calendar', 'action_calendar_event', context)
```

Comment:
use self.pool['ir.actions.act_window'] instead of self.pool.get('ir.actions.act_window')